source/user-manual/capabilities/file-integrity/fim-fields-rule-mapping.rst

```diff
@@ -0,0 +1,145 @@
+.. Copyright (C) 2021 Wazuh, Inc.
+
+Fim fields rule mapping
```

Comment:
```suggestion
FIM fields rule mapping
```

Reply:
Done in [commit](https://github.com/wazuh/wazuh-documentation/pull/3913/commits/d5ce0fcadd5abf8bbb40c82c8ccbba1b333c2572)